### PR TITLE
add: multiselect

### PR DESCRIPTION
**Pull request by GustavoUT22**
Check it out!

### CODE REVIEW

File: src/ui/InputSearch/InputSearch.styles.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import styled from 'styled-components'
```
Comment (CrhistianT7) [inline src/ui/InputSearch/InputSearch.styles.ts:1]: add a breakline here

File: src/ui/InputSearch/InputSearch.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import React, { forwardRef } from 'react'
```
Comment (CrhistianT7) [inline src/ui/InputSearch/InputSearch.tsx:1]: Remove React import from here and add a breakline

File: src/ui/MultiSelect/MultiSelect.styles.ts [hunk 3]
```diff
@@ -0,0 +1,109 @@
+import { FiSearch } from 'react-icons/fi'
+import styled from 'styled-components'
+import { colors } from 'styles/colors'
```
Comment (CrhistianT7) [inline src/ui/MultiSelect/MultiSelect.styles.ts:3]: order imports here

File: src/ui/MultiSelect/MultiSelect.tsx [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+      (ele) => ele.key !== service.key
+    )
+    setSelectedServices(filteredArray)
+    const updatedServices = new Set(selectedServicesSet)
+    updatedServices.delete(service.key)
+    setSelectedServicesSet(updatedServices)
+    onChange(filteredArray)
+  }
+
+  const handleIsOpen = () => {
```
Comment (CrhistianT7) [inline src/ui/MultiSelect/MultiSelect.tsx:79]: remove this function and send directly the SetIsOpen to onclick event

File: src/ui/InputSearch/InputSearch.styles.ts (hunk 1 above)
Comment (GustavoUT22) [reply src/ui/InputSearch/InputSearch.styles.ts:1]: done

File: src/ui/InputSearch/InputSearch.tsx (hunk 2 above)
Comment (GustavoUT22) [reply src/ui/InputSearch/InputSearch.tsx:1]: done !

File: src/ui/MultiSelect/MultiSelect.styles.ts (hunk 3 above)
Comment (GustavoUT22) [reply src/ui/MultiSelect/MultiSelect.styles.ts:3]: done

File: src/ui/MultiSelect/MultiSelect.tsx (hunk 4 above)
Comment (GustavoUT22) [reply src/ui/MultiSelect/MultiSelect.tsx:79]: done

**Review by CrhistianT7 — APPROVED** [review]
Good Job!!!

**Pull request merged**